<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Socket Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>IP:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ip_option">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>port 1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="port_1">
        <property name="maximum">
         <number>1000000000</number>
        </property>
        <property name="value">
         <number>5000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>port 2:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="port_2">
        <property name="maximum">
         <number>1000000000</number>
        </property>
        <property name="value">
         <number>6000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connect_button">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="disconnect_button">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="reset_button">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="stop_auto_mode_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="auto_mode_button">
            <property name="text">
             <string>Auto Mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="auto_mode_startup_option">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Auto startup</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="wafer_table">
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <column>
           <property name="text">
            <string>Wafer ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Position</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="test_button_layout"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QComboBox" name="mv_w_id_option"/>
          </item>
          <item>
           <widget class="QComboBox" name="to_option">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>A</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>C</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>D</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>E</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>F</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CassA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CassB</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mv_w_button">
            <property name="text">
             <string>Move Wafer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="virtual_reply_button">
            <property name="text">
             <string>Virtual Reply</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="text_option"/>
          </item>
          <item>
           <widget class="QPushButton" name="send_button">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clear_button_1">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="console_1"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="data_limit_table">
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <row>
           <property name="text">
            <string>B</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>D</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>E</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>F</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Win Out</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Win In</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Time</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Steps:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="mv_counter">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="clear_button_2">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="console_2"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clear_button_1</sender>
   <signal>clicked()</signal>
   <receiver>console_1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_button_2</sender>
   <signal>clicked()</signal>
   <receiver>console_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
